cves/CVE-2019-6975.yml

```diff
@@ -49,7 +49,10 @@ description_instructions: |
 
   Your target audience is people just like you before you took any course in
   security
-description:
+description: |
+  Excessive memory usage can be caused by calling django.utils.numberformat.format()
+  on a Decimal. The root cause is a call to '{:f}'.format() with a Decimal with a large
```

Comment:
I wouldn't reference direct parts of Django's spec, keep it in simple language that doesn't need a reference document to be understood.

Reply:
You're right. I simplified it.